app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -10,8 +10,12 @@ def hawkular_client
       @hawkular_entrypoint, @hawkular_credentials, @hawkular_options)
   end
 
+  # may be nil
+  def hawkular_endpoint
```

Comment:
@cben it seems that having two similar methods as `hawkular_endpoint` and `hawkular_entrypoint` is confusing. Any way to improve this part?